modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/js/input_localized/InputLocalized.tsx

```diff
@@ -44,8 +44,8 @@ function InputLocalized({
 	componentId,
 	defaultLanguageId,
 	editorConfig,
-	fieldPrefix = '',
-	fieldPrefixSeparator = '',
+	fieldPrefix,
```

Comment:
Ooh, I honestly thought that default props kick in on `null`. TIL!

Reply:
Ya, I feel it'd be nice if it would since at least I don't think I've ever never needed `null` over the fallback 🙃